### ISSUE TITLE
https://peer2profit.com/ doesn't exist anymore.

### ISSUE THREAD

**Issue by lyenliang** [OlivierGaland/CashFactory#52]
Today when I visited https://peer2profit.com/, I got a `DNS_PROBE_POSSIBLE` error.

This seems that peer2profit doesn't work anymore.

Can anyone else successfully visit https://peer2profit.com/ ?

My peer2profit docker container keeps generating the following error messages:

```
2023-10-19T15:45:59Z | Waiting configuration.
2023-10-19T15:46:00Z | Configuration request.
2023-10-19T15:46:00Z | Getting config is error. NetworkError: 2 : Connection closed
2023-10-19T15:46:00Z | Error while getting of configuration. Repeat.
2023-10-19T15:47:20Z | Waiting configuration.
2023-10-19T15:47:22Z | Configuration request.
2023-10-19T15:47:22Z | Getting config is error. NetworkError: 2 : Connection closed
2023-10-19T15:47:22Z | Error while getting of configuration. Repeat.
2023-10-19T15:48:42Z | Waiting configuration.
2023-10-19T15:48:43Z | Configuration request.
2023-10-19T15:48:43Z | Getting config is error. NetworkError: 2 : Connection closed
2023-10-19T15:48:43Z | Error while getting of configuration. Repeat.
2023-10-19T15:50:03Z | Waiting configuration.
2023-10-19T15:50:04Z | Configuration request.
```

**Comment by RocketDevil** [comment]
So in the past like 1,5 years you never did read anything about p2p and how it will work and how u register devices in the future? Its not like there was a big banner on the top, saying that the site will be closed and telegram connection will be the only way to work... :D 

maybe u wanna read the official [Telegram works with your Site Login Details](https://t.me/peer2profit_app_bot?start=1649167887624c4e0f85816) 

It only works atm with already Registered Devices to earn Money, i think but I'm not 100% sure about that, so u have to try to and register a new account if u never had one...

**Comment by OlivierGaland** [comment]
Hello,

The website is down since a few monthes but service seems to continue. It seems the docker container is no more working in my stack, so I guess the solution is running it as standalone. I will have a look if there a workaround for docker.

The API is now a bot in telegram, I received a mail about how to register like 1 year ago, but I do not have the link to the correct channel (so be careful if you search it on the web, prefer have a look in your mailbox first).

I can confirm the payout is working.

![image](https://github.com/OlivierGaland/CashFactory/assets/26048157/bf83b69e-664a-4970-baca-5b3c248f45db)

**Comment by OlivierGaland** [comment]
Issue is fixed and provided in release v1.7
